website/source/docs/providers/aws/r/alb_listener.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: "aws"
+page_title: "AWS: aws_alb_listener"
+sidebar_current: "docs-aws-resource-alb-listener"
+description: |-
+  Provides an Application Load Balancer Listener resource.
+---
+
+# aws\_alb\_listener
+
+Provides an Application Load Balancer Listener resource.
+
+## Example Usage
+
+```
+# Create a new load balancer
+resource "aws_alb" "front_end" {
+  # Other parameters...
+}
+
+resource "aws_alb_target_group" "front_end" {
+  # Other parameters...
+}
+
+resource "aws_alb_listener" "front_end" {
+   load_balancer_arn = "${aws_alb.front_end.id}"
```

Comment:
I realize that in our case, the ID and the ARN of the ALB are the same, but what do you think about exposing the ARN on the ALB resource, and then referencing that here? 

I think that, while it's kind of a minor detail for us, it makes more sense to a user use a config like this:

load_balancer_**arn** = "${aws_alb.front_end.**arn**}"

over this

load_balancer_**arn** = "${aws_alb.front_end.**id**}"